### PR TITLE
#90 more docs

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**

### CODE REVIEW

File: gui/doc/selector.md [hunk 1]
```diff
@@ -28,7 +28,59 @@ You can create a selector on a series of strings:
...
+    === "Markdown"
+
+        ```
+        <|{sel}|selector|lov={[("id1", "Label 1"), ("id2", Icon("/images/icon.png", "Label 2"),("id3", "Label 3")]}|>
+        ```
+  
+    === "HTML"
+
+        ```html
+        <taipy:selector lov="{[('id1', 'Label 1'), ('id2', Icon('/images/icon.png', 'Label 2'),('id3', 'Label 3')]}">{sel}</taipy:selector>
```
Comment (dinhlongviolin1) [inline gui/doc/selector.md]: you might want to add examples for self closing tags

<taipy:selector value="{sel}" lov="{[('id1', 'Label 1'), ('id2', Icon('/images/icon.png', 'Label 2'),('id3', 'Label 3')]}" />

**Review by dinhlongviolin1 — APPROVED** [review]
Excellent work! Just have a single comment

File: gui/doc/dialog.md [hunk 2]
```diff
@@ -98,7 +94,7 @@ The content of the dialog can be specified as an existing page name using the _p
 
 ### Dialog with partial
 
-The content of the dialog can be specified as a _Partial_ instance using the _partial_ property.
+The content of the dialog can be specified as a Partial^ instance using the _partial_ property.
```
Comment (FabienLelaquais) [inline gui/doc/dialog.md]: That would be `Partial^` (with backticks)

File: gui/doc/navbar.md [hunk 3]
```diff
@@ -4,10 +4,40 @@ This control is implemented as a list of links.
 
 ## Usage
 
-### Simple
+### Defining a default navbar
 
-<code><|navbar|></code>
+The list of pages is used to build the navbar.
```
Comment (FabienLelaquais) [inline gui/doc/navbar.md]: You mean... the list of all pages registered in the Gui instance?

File: gui/doc/navbar.md [hunk 4]
```diff
@@ -4,10 +4,40 @@ This control is implemented as a list of links.
...
+    === "HTML"
+
+        ```html
+        <taipy:navbar></taipy:navbar>
+        ```
+
+
+### Defining a custom navbar
+
+The _lov_ property is used to define the list of element visible in the navbar.
```
Comment (FabienLelaquais) [inline gui/doc/navbar.md]: the list of element += 's that are displayed'

File: gui/doc/pane.md [hunk 5]
```diff
@@ -1,21 +1,122 @@
 A side pane.
 
-TODO: Complete
+Pane allows to show some content over the current page.
```
Comment (FabienLelaquais) [inline gui/doc/pane.md]: "over" -> "on top of"

File: gui/doc/pane.md [hunk 6]
```diff
@@ -1,21 +1,122 @@
...
+        <|{show}|pane|anchor=left|>
+        ```
+  
+    === "HTML"
+
+        ```html
+        <taipy:pane anchor="left">{show}</taipy:pane>
+        ```
+
+### Showing the pane beside the page content
```
Comment (FabienLelaquais) [inline gui/doc/pane.md:44]: Is it 'beside' or 'besides'? I never really know...

File: gui/doc/pane.md [hunk 7]
```diff
@@ -1,21 +1,122 @@
...
+        <|{show}|pane|persistent|>
+        ```
+  
+    === "HTML"
+
+        ```html
+        <taipy:pane persistent="True">{show}</taipy:pane>
+        ```
+
+### Pane as block
```
Comment (FabienLelaquais) [inline gui/doc/pane.md]: Pane as a block element?

File: gui/doc/pane.md [hunk 8]
```diff
@@ -1,21 +1,122 @@
...
+  
+    === "HTML"
+
+        ```html
+        <taipy:pane page="page_name">{show}</taipy:pane>
+        ```
+
+### Pane with partial
+
+The content of the pane can be specified as a Partial^ instance using the _partial_ property.
```
Comment (FabienLelaquais) [inline gui/doc/pane.md]: -> `Partial^`

File: gui/doc/part.md [hunk 9]
```diff
@@ -1,17 +1,55 @@
 Displays its children in a block.
 
-Part controls can be simplified by specifying no component. TODO: explain.
+The Part control is used to group controls.
```
Comment (FabienLelaquais) [inline gui/doc/part.md]: Use `part` instead of Part:
"The `part` control is used to group controls in a single element."

File: gui/doc/part.md [hunk 10]
```diff
@@ -1,17 +1,55 @@
 Displays its children in a block.
 
-Part controls can be simplified by specifying no component. TODO: explain.
+The Part control is used to group controls. 
+This allows to show or hide them in one action and to display them in one Layout^ cell.
```
Comment (FabienLelaquais) [inline gui/doc/part.md]: "display them in one Layout^ cell"
->
"be placed as a unique element in a `Layout^`  cell."

File: gui/doc/part.md [hunk 11]
```diff
@@ -1,17 +1,55 @@
...
+Part controls can be simplified by not specifying the part keyword.
```
Comment (FabienLelaquais) [inline gui/doc/part.md]: "There is a simplified Markdown syntax to create a `part`, where the element name is optional:
   - `<|` just before the en of the line indicates the beginning of a part element;
   - `|>` at eh beginning of a line indicated the end of the part definition."

File: gui/doc/part.md [hunk 12]
```diff
@@ -1,17 +1,55 @@
...
+
+        ```html
+        <taipy:part>
+            ...
+            <taipy:text>{Some Content}</taipy:text>
+            ...
+        </taipy:part>
+        ```
+
+### Showing/Hiding controls
```
Comment (FabienLelaquais) [inline gui/doc/part.md]: Showing and hiding

File: gui/doc/part.md [hunk 13]
```diff
@@ -1,17 +1,55 @@
...
+  
+    === "HTML"
+
+        ```html
+        <taipy:part render="False">
+            ...
+            <taipy:text>{Some Content}</taipy:text>
+            ...
+        </taipy:part>
+        ```
```
Comment (FabienLelaquais) [inline gui/doc/part.md:59]: If the _render_ property is bound to a Boolean value, the `part` will show or hide its elements according to the value of the bound variable.

File: gui/doc/selector.md [hunk 14]
```diff
@@ -28,7 +28,59 @@ You can create a selector on a series of strings:
...
+    === "HTML"
+
+        ```html
+        <taipy:selector lov="Item 1;Item 2;Item 3" dropdown="True">{value}</taipy:selector>
+        ```
+
+
+### Display with filter and multiple selection
+
+!!! example "Page content"
```
Comment (FabienLelaquais) [inline gui/doc/selector.md:52]: You can add a filter input field that lets you display only strings that match the filter value.

File: gui/doc/slider.md [hunk 15]
```diff
@@ -6,7 +6,7 @@ If the _lov_ property is used, then the slider can be used to select a value amo
 
 ## Usage
 
-### Simple
+### Selecting a value between 0 and 100
```
Comment (FabienLelaquais) [inline gui/doc/slider.md:9]: Is it 0 or 1 ?

File: gui/doc/status.md [hunk 16]
```diff
@@ -3,18 +3,41 @@ Displays a status or a list of statuses.
...
 
-<code><|{value}|status|></code>
+!!! example "Page content"
 
-### Advanced
+    === "Markdown"
 
-<code><|{value}|status|></code>
+        ```
+        <|{value}|status|>
```
Comment (FabienLelaquais) [inline gui/doc/status.md:20]: What would 'value' be, in Python?

File: gui/doc/status.md [hunk 17]
```diff
@@ -3,18 +3,41 @@ Displays a status or a list of statuses.
...
 
-or with properties
+        ```html
+        <taipy:status>{value}</taipy:status>
+        ```
 
-<code><|{value}|status|properties={properties}|></code>
+### Prevent status dismiss
+
+!!! example "Page content"
```
Comment (FabienLelaquais) [inline gui/doc/status.md:33]: If you don't want the user to be allowed to dismiss the displayed statuses, you can use the `without_close` property.

**Review by FabienLelaquais — APPROVED** [review]
Minor things that I think clarify even better.
Thanks! Great job.

File: gui/doc/pane.md (hunk 6 above)
Comment (FredLL-Avaiga) [reply gui/doc/pane.md:44]: I check with my personal reference: beside it is in this context

File: gui/doc/slider.md (hunk 15 above)
Comment (FredLL-Avaiga) [reply gui/doc/slider.md:9]: 0

File: gui/doc/status.md (hunk 16 above)
Comment (FredLL-Avaiga) [reply gui/doc/status.md:20]: I described it at the top of "Usage"

**Comment by FredLL-Avaiga** [comment]
> Minor things that I think clarify even better. Thanks! Great job.

1 question left :-)

File: gui/doc/pane.md (hunk 6 above)
Comment (FabienLelaquais) [reply gui/doc/pane.md:44]: Yeah that's what I thought but Grammarly somtimes(?) complains, even in those cases... forget it. Thanks

**Pull request merged**